screens/Home/ProductsScreen.tsx

```diff
@@ -9,11 +9,11 @@ import { Product } from '../../models/products.model';
 
 type ParamList = { Params: { search: string } };
 
-const RenderProducts = (products: Product[]) => {
+const RenderProducts = (products: Product[], search: string) => {
 
   const navigation = useNavigation();
   return (
-    products?.map((product, index) => {
+    products?.filter(productItems => productItems.category === search || productItems.title.toLowerCase().includes(search.toLowerCase()) || productItems.description.toLowerCase().includes(search.toLowerCase()) )?.map((product, index) => {
```

Comment:
Nice, I like that you can search on multiple fields at once.

Reply:
Thank You.